apps/test-suite/tests/KeepAwake.js

```diff
@@ -0,0 +1,23 @@
+import * as KeepAwake from 'expo-keep-awake';
+
+export const name = 'KeepAwake';
+
+export async function test(
+  { it, describe, beforeAll, jasmine, afterAll, expect, afterEach, beforeEach },
+  { setPortalChild, cleanupPortal }
+) {
+  describe(name, () => {
+    it(`keeps the screen on`, async () => {
```

Comment:
Add afterAll that calls deactivateKeepAwake and deactivateKeepAwake('test-tag') so we clean up everything for sure, regardless of the order in which the tests run or whether they succeed.